build.sbt

```diff
@@ -31,11 +31,11 @@ inThisBuild(
 addCommandAlias("fmt", "all scalafmtSbt scalafmt test:scalafmt")
 addCommandAlias("check", "all scalafmtSbtCheck scalafmtCheck test:scalafmtCheck")
 
-val zioVersion            = "2.0.2"
-val zioNioVersion         = "2.0.0"
-val zioConfigVersion      = "3.0.2"
+val zioVersion            = "2.0.20"
+val zioNioVersion         = "2.0.1"
+val zioConfigVersion      = "3.0.7"
 val zioInteropCatsVersion = "22.0.0.0"
-val akkaActorTypedVersion = "2.6.19"
+val akkaActorTypedVersion = "2.8.0"
```

Comment:
We have to stay on the last non-BSL version. Please revert this.

Reply:
:grimacing: Good catch @vigoo, `2.6.19` does unfortunately not support Scala 3 so I'll have to migrate to Pekko. Do you see any problems with that? If not it is a pretty easy migration to make.

Reply:
Also added comment so it won't be upgraded by mistake again